elixir/config/dev.exs

```diff
@@ -69,7 +69,7 @@ config :phoenix_live_reload, :dirs, [
 config :web, Web.Plugs.SecureHeaders,
   csp_policy: [
     "default-src 'self' 'nonce-${nonce}' https://api-js.mixpanel.com",
-    "img-src 'self' data: https://www.gravatar.com https://track.hubspot.com",
+    "img-src 'self' data: https://www.gravatar.com https://track.hubspot.com https://www.firezone.dev",
```

Comment:
Why do we need CSP updates for emails?

Reply:
We pull the Firezone logo from the website and I wanted to be able to make sure the logo loaded as expected.

Reply:
But CSP doesn't apply to emails or I'm missing something?

Reply:
I believe it does for the Swoosh mailbox view.

Reply:
### Without `https://www.firezone.dev` added to the CSP header
<img width="1114" alt="Screenshot 2024-09-10 at 2 54 05 PM" src="https://github.com/user-attachments/assets/0fa4f9e9-c77b-4ddd-9d9c-d72a73f86d0f">

<hr />

### With `https://www.firezone.dev` added to the CSP header
<img width="1020" alt="Screenshot 2024-09-10 at 2 55 10 PM" src="https://github.com/user-attachments/assets/dc1b29fa-d813-40c3-b432-ba49635500a9">

Reply:
@bmanifold ahh, yeah, you are right - swoosh in dev mode needs that. Thanks!